Rollbar/RollbarNotifier.m

```diff
@@ -331,26 +341,32 @@ - (BOOL)sendPayload:(NSData*)payload {
     [request setValue:self.configuration.accessToken forHTTPHeaderField:@"X-Rollbar-Access-Token"];
     [request setHTTPBody:payload];
     
-    NSError *error;
-    NSHTTPURLResponse *response;
+//    NSError *error;
+//    NSHTTPURLResponse *response;
+//    NSData *data = [NSURLConnection sendSynchronousRequest:request returningResponse:&response error:&error];
     
-    NSData *data = [NSURLConnection sendSynchronousRequest:request returningResponse:&response error:&error];
+    dispatch_semaphore_t sem = dispatch_semaphore_create(0);
     
-    if (error) {
-        RollbarLog(@"There was an error reporting to Rollbar");
-        RollbarLog(@"Error: %@", [error localizedDescription]);
-    } else {
-        NSHTTPURLResponse* httpResponse = (NSHTTPURLResponse*)response;
-        if ([httpResponse statusCode] == 200) {
-            RollbarLog(@"Success");
-            return YES;
+    __block BOOL result = NO;
+    NSURLSessionDataTask *dataTask = [[NSURLSession sharedSession] dataTaskWithRequest:request completionHandler:^(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error) {
```

Comment:
We need to decide on our lowest supported iOS version, it is probably safe to go 7.0+ now which is what this requires, but we need to be sure to document that somewhere.

Reply:
@rokob should I put this in README?

Reply:
Changed to use the deprecated codes only in version below 7.